@navikt/core/react/src/form/form-progress/FormProgress.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import { Meta, StoryFn } from "@storybook/react";
+import React, { useState } from "react";
+import VStack from "../../layout/stack/VStack";
+import FormProgress, { FormProgressProps } from "./FormProgress";
+
+export default {
+  title: "ds-react/FormProgress",
+  component: FormProgress,
+  decorators: [
+    (story) => (
+      <div style={{ width: 300, maxWidth: "100%", margin: "0 auto" }}>
+        {story()}
+      </div>
+    ),
+  ],
+  parameters: { layout: "padded", chromatic: { disable: true } },
```

Comment:
Er ikke padded default? 🤔 
<img width="701" alt="Screenshot 2024-05-08 at 15 40 20" src="https://github.com/navikt/aksel/assets/26967723/83cff107-4cee-4527-8620-d09a7a3d03a4">
https://storybook.js.org/docs/configure/story-layout

Reply:
Settes til `centered` i preview.jsx. Satte den til `padded` her for å kunne teste responsivitet, men trengs for så vidt ikke.